merkletree/pad_test.go

```diff
@@ -241,14 +241,14 @@ func TestPoliciesChange(t *testing.T) {
 }
 
 func TestTB(t *testing.T) {
```

Comment:
Yes, why not. It makes sense (https://github.com/coniks-sys/coniks-go/pull/54/commits/04cf99ad1e002027dfe6feb92dba7ad444543dbb)